### PR TITLE
#590: Fix Asciidoc files in documentation causing errors

### PR DESCRIPTION
**Pull request by alfeilex**
Fixes: #590

The main error occured by a missing line break of `|==` in `projects.adoc`. It looks like it traverse until `Licence.adoc`.

Further some links were incorrectly set to other files. E.g `link:setups.asciidoc` instead of `link:setups.adoc`. 

Finally some refactoring **suggested by intellij** was made, because `= <title>` main title should be before `[toc]`. There are some other adoc files that have that, but it is not critical error that is also not recognized by `asciidoctor-maven-plugin`

### CODE REVIEW

File: documentation/LICENSE.adoc [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-= License
-
 The product `IDEasy` is licensed under the following terms.
 
 Binaries of this product have been made available to you by https://devonfw.com[devonfw] under the https://github.com/devonfw/ide/blob/master/LICENSE[Apache License 2.0].
 
 The documentation of this product is licensed under the terms of the http://creativecommons.org/licenses/by-nd/4.0/[Creative Commons License (Attribution-No Derivatives 4.0 International)].
 
-All of the source code to this product is available under licenses which are both free and open source.
+All the source code to this product is available under licenses which are both free and open source.
```
Comment (jan-vcapgemini) [inline documentation/LICENSE.adoc:11]:
```suggestion
The source code for this product is available under free and open-source licenses.
```

**Review by jan-vcapgemini — APPROVED** [review]
LGTM. I've added a small suggestion.

File: documentation/projects.adoc [hunk 2]
```diff
@@ -20,11 +17,11 @@ import=eclipse,intellij
...
 ----
 
- .Variables of project import
-[options="header"]
-
-|=== |*Variable*|*Value*|*Meaning*
-|`path`|e.g. `myproject`, will clone into `${WORKSPACE_PATH}/myproject`|(required) Path into which the projects is cloned.
+.Variables of project import
+[options="header",cols="3"]
+|===
```
Comment (alfeilex) [inline documentation/projects.adoc:22]: This is the main spot that caused the error in `projects.adoc` and `License.adoc`

File: documentation/LICENSE.adoc [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-= License
-
 The product `IDEasy` is licensed under the following terms.
 
 Binaries of this product have been made available to you by https://devonfw.com[devonfw] under the https://github.com/devonfw/ide/blob/master/LICENSE[Apache License 2.0].
 
 The documentation of this product is licensed under the terms of the http://creativecommons.org/licenses/by-nd/4.0/[Creative Commons License (Attribution-No Derivatives 4.0 International)].
 
-All of the source code to this product is available under licenses which are both free and open source.
+The source code for this product is available under free and open-source licenses.
```
Comment (hohwille) [inline documentation/LICENSE.adoc:11]:
```suggestion
The source code of this product is available under free and open-source licenses.
```

**Review by hohwille — APPROVED** [review]

**Comment by hohwille** [comment]
> [INFO] --- asciidoctor-maven-plugin:1.6.0:process-asciidoc (generate-doc) @ ide-doc ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 19 resources
[INFO] Rendered /home/runner/work/IDEasy/IDEasy/documentation/target/asciidoc/IDEasy.adoc

So nice. Thanks @alfeilex 👍

**Pull request merged**